coder-sdk/client_test.go

```diff
@@ -103,6 +106,7 @@ func TestPasswordAuthentication(t *testing.T) {
 		Password:   "coder4all",
 	})
 	require.NoError(t, err, "failed to create Client")
+	require.Equal(t, "g4mtIPUaKt-pPl9Q0xmgKs7acSypHt4Jf", client.Token(), "expected token to match")
```

Comment:
Should store this token in a variable or constant.

Reply:
I kinda went back and forth on it, using constants could obscure test problems and also obscure what the tests are testing, but I don't feel strongly about it so will switch over to constants